pip/flatpak-pip-generator

```diff
@@ -169,11 +150,65 @@ with open(requirements_file, 'r') as req_file:
     use_hash = '--hash=' in req_file.read()
 
 python_version = '2' if opts.python2 else '3'
+devnull = subprocess.DEVNULL
+
+# Prefer to use Sdk as they bundle `git`.
+possible_runtimes = [
```

Comment:
Not a big fan of this. We could import libflatpak which adds a dependency but makes it more robust...

Will have to think on that.

Reply:
Me neither. Originally I wanted to use `org.freedesktop.Platform//19.08` as a default, but that requires maintenance both from the users an upstream. There is also the issue that if there are multiple branchs of the same runtime, `flatpak info --show-runtime $runtime` won't return anything.

I don't know if all distributions have access to libflatpak, at least silverblue provides it in the base image. I would consider this a regression if users need to install dependencies outside of python and there is no uniform way of providing them. I was thinking that in the future flatpak-builder-tools could be shipped as a flatpak.